.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -18,6 +18,31 @@ body:
         - label: I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)
           required: true
 
+  - type: dropdown
+    id: install
+    attributes:
+      label: How did you install golangci-lint?
+      options:
+        - I don't know
+        - Official GitHub Action
+        - Official binary
+        - Brew
+        - MacPorts
+        - Chocolatey
+        - Scoop
+        - Docker
+        - go install
+        - Tools pattern
+        - go tool
```

You mean tools.go pattern? The one used before 1.24 go tool

If so, it should be clearer.


Here is a suggestion, that could be improved of course

```suggestion
        - tools.go pattern (pre Go 1.24 way)
        - go tool (Go 1.24+)
```

---

The name of the pattern is `Tools pattern` not `tools.go pattern`.